ansible/playbooks/submitty_install.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Install Submitty
```

Comment:
Add a comment at the beginning of the playbook:

- Example playbook
- You can customize the variable to the app_submitty role by looking the default/main.yml list of variables in the role.

And add a new line at the end of the file.

Reply:
We should just have submitty version, remove all others vars and make comment saying here we can override the defaults.